src/SFML/Window/OSX/HIDInputManager.mm

```diff
@@ -30,6 +30,9 @@
 #include <SFML/System/Err.hpp>
 #include <AppKit/AppKit.h>
 
+static const sf::Uint8 UnknownVirtualCode = 0xff;
+static const bool IsIsoKeyboard = (KBGetLayoutType(LMGetKbdType()) == kKeyboardISO);
```

Comment:
I hesitated when writing this because, as far as I know, guarantee of internal linkage for anonymous namespaces comes with C++11 (not C++03), but on the other hand, anonymous namespaces are used elsewhere in the code base. So it makes sense to have an anonymous namespace here.